### DISCUSSION TITLE
Can I Generate Samples Without Tumors?

### DISCUSSION THREAD

**Discussion by juandtap** [mobaidoctor/med-ddpm#40] [Q&A] [answered]
Hello friends,
I am working on my graduation project, and I want to create a dataset with a control group. I already have the BraTS 2024 dataset, but now I want to include healthy samples. Can I generate samples without tumors? What parameters do I need to change to achieve that?

Thanks for your help.

**Comment by mobaidoctor** [comment, marked as answer]
Hi @juandtap, thank you for your interest in our work and for your inquiry. Yes, you can generate healthy samples by providing only brain labels without including any tumor mask labels. You can refer to the label values in utils/dtypes_brats.py.
